<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuromagSetupWidgetClass</class>
 <widget class="QWidget" name="NeuromagSetupWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTServerSetupClass</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_pref">
      <attribute name="title">
       <string>Preferences</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="m_qGridLayout_main">
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="m_qLabelNeuromagImage">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../neuromag.qrc">:/images/neuromag.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="1" rowspan="2" colspan="3">
          <widget class="QGroupBox" name="m_qGroupBox_Channels">
           <property name="title">
            <string>Fiff Channel</string>
           </property>
           <layout class="QGridLayout" name="m_qGridLayout_Channel">
            <item row="0" column="0">
             <widget class="QColumnView" name="m_qColumnView_Channels"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="m_qPushButton_About">
           <property name="text">
            <string>About</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <spacer name="m_qHorizontalSpacer_About">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" rowspan="2" colspan="3">
          <widget class="QGroupBox" name="m_qGroupBox_Information">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Information</string>
           </property>
           <layout class="QGridLayout" name="m_qGridLayout_Information">
            <item row="0" column="0">
             <widget class="QTextBrowser" name="m_qTextBrowser_Information"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="m_qHorizontalSpacerTitle">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="m_qGroupBox_MneRtClientProperties">
           <property name="title">
            <string>Neuromag Preferences</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="1">
             <widget class="QLineEdit" name="m_qLineEdit_BufferSize">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="m_qLabel_BufferSize">
              <property name="text">
               <string>Buffer [samples]:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="m_qLabel_sps">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="m_qLabel_sps_description">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Sampling Rate [sps]:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="m_qGroupBox_PluginProperties">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Plugin Properties</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="m_qGridLayout_Properties">
            <item row="1" column="0">
             <widget class="QLabel" name="m_qLabel_Downsampling">
              <property name="text">
               <string>Downsampling: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="m_qSpinBox_Downsampling">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="suffix">
               <string>x</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="m_qVerticalSpacer_Headline">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_con">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QGroupBox" name="m_qGroupBox_Send">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Send Command Line Instruction</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="m_qLabel_Instruction">
              <property name="text">
               <string>Instruction:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="m_qLineEdit_SendCLI"/>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="m_qPushButton_SendCLI">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="m_qGroupBox_rtServerInfo">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Host (mne_rt_server) Information</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="m_qLabel_Ip">
              <property name="text">
               <string>IP Address:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="m_qLabel_ConnectionStatusDescr">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="m_qPushButton_Connect">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="m_qLineEdit_Ip"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="m_qLabel_ConnectionStatus">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>not Connected</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2" rowspan="3">
          <widget class="QGroupBox" name="m_qGroupBox_log">
           <property name="title">
            <string>Log</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0" rowspan="2">
             <widget class="QTextBrowser" name="m_qTextBrowser_ServerMessage"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="1" rowspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../neuromag.qrc"/>
 </resources>
 <connections/>
</ui>
